File: scripts/commands/generic/setWaypointName.py
import sys

#Args: name
def execute(objManager, player, target, args):
	ghost = player.getPlayerObject()
	if ghost is None:
		return
	
	waypoint = ghost.getWaypoint(target.getObjectId())
	if waypoint is None:
		return
	
	waypoint.setName(args)
	ghost.updateWaypoint(waypoint)
	return

File: scripts/commands/generic/setWaypointName.js
var execute = function(objManager, player, target, args) {
	var ghost = player.getPlayerObject();
	var waypoint;
	
	if(ghost == null || target == null) {
		return;
	}
	
	waypoint = ghost.getWaypoint(target.getObjectId());
	
	if(waypoint == null) {
		return;
	}
	
	waypoint.setName(args);
	
	ghost.updateWaypoint(waypoint);
}